docs/reference/commandline/rm.md

```diff
@@ -29,40 +29,68 @@ Options:
 
 ## Examples
 
-    $ docker rm /redis
-    /redis
+### Remove a container
 
 This will remove the container referenced under the link
 `/redis`.
 
-    $ docker rm --link /webapp/redis
-    /webapp/redis
+```bash
+$ docker rm /redis
+
+/redis
+```
+
+### Remove a container and its links
 
 This will remove the underlying link between `/webapp` and the `/redis`
 containers removing all network communication.
+```bash
+$ docker rm --link /webapp/redis
+
+/webapp/redis
+```
+
+### Force-remove a running container
```

Comment:
For a follow up; we may want to express the dangers of using `-f`, as it not only forces removal of a container is still running, but also can result in layers being left behind if they fail to be removed